recipe/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
```

Comment:
As [`tslearn` uses `setuptools`]( https://github.com/tslearn-team/tslearn/blob/5568c026db4b4380b99095827e0573a8f55a81f0/setup.py#L1 ), we need to add `setuptools` to `requirements/host` as done here